src/tests/ftest/server/metadata.py

```diff
@@ -236,24 +246,21 @@ def run_dummy_metadata_workload(self, duration=150):
 
         return True
 
-    def test_metadata_fillup(self):
-        """JIRA ID: DAOS-1512.
-
-        Test Description:
-            Test to verify no IO happens after metadata is full.
+    def metadata_fillup(self, svc_ops_enabled=True):
+        """Run test to verify no IO happens after metadata is full.
 
-        Use Cases:
-            ?
+        Args:
+            svc_ops_enabled (bool): Pool create properties svc_ops_enabled, default to True.
 
-        :avocado: tags=all,full_regression
-        :avocado: tags=hw,large
-        :avocado: tags=server,metadata
-        :avocado: tags=ObjectMetadata,test_metadata_fillup
         """
-        self.create_pool()
-        svc_ops_entry_age = self.pool.get_property("svc_ops_entry_age")
-        if not self.run_dummy_metadata_workload(duration=svc_ops_entry_age):
-            self.fail("failed to run dummy metadata workload")
+        self.log_step("1. Create pool with properties svc_ops_enabled: {}".format(svc_ops_enabled))
```

Comment:
We shouldn't be adding numbers when using `log_step` because `log_step` already adds numbers

Reply:
The reason to add 1. 2. 3.... for the step is sync with the test steps in the testcase description.

Reply:
But this output is confusing
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14145/12/artifact/Functional%20Hardware%20Large/server/metadata.py/steps.log
```
Step 5: 1. Create pool with properties svc_ops_enabled: True [elapsed since last step: 0.00s]
Step 6: 2. sustained container creates: to no space and beyond. [elapsed since last step: 66.16s]
Step 7: 3. Verify number of container within limit. [elapsed since last step: 114.49s]
Step 8: 4. Cleaning up 500 containers after pool is full. [elapsed since last step: 0.00s]
```

Reply:
I think it's clear. the global Step 4, and the testcase step 1.

Reply:
@phender opinion?